### PR TITLE
Windows: Add missing parameter types and doc fixes

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- Adding missing parameter types
- Various documentation fixes

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
Windows modules

### CODE REVIEW

File: lib/ansible/modules/windows/win_copy.py [hunk 1]
```diff
@@ -21,19 +21,21 @@
   content:
     description:
     - When used instead of C(src), sets the contents of a file directly to the
-      specified value. This is for simple values, for anything complex or with
-      formatting please switch to the template module.
+      specified value.
+    - This is for simple values, for anything complex or with formatting please
+      switch to the template module.
```
Comment (acozine) [inline lib/ansible/modules/windows/win_copy.py]:
```suggestion
      switch to the M(template) module.
```

File: lib/ansible/modules/windows/win_credential.py [hunk 2]
```diff
@@ -120,13 +113,9 @@
...
       providers can access the secret.
-    required: True
     type: str
-    choices:
-    - domain_password
-    - domain_certificate
-    - generic_password
-    - generic_certificate
+    required: true
+    choices: [ domain_password, domain_certificate, generic_password, generic_certificate ]
```
Comment (acozine) [inline lib/ansible/modules/windows/win_credential.py]:
```suggestion
    choices: [ domain_certificate, domain_password, generic_certificate, generic_password ]
```

File: lib/ansible/modules/windows/win_domain_computer.py [hunk 3]
```diff
@@ -22,57 +20,64 @@
...
-        operating systems compatibility. The LDAP display name
-        (ldapDisplayName) for this property is sAMAccountName. If ommitted the
-        value is the same as C(name).
-        Note. All computer SAMAccountNames needs to end with a $.
+        computer.
+      - It maximum is 256 characters, 15 is advised for older
+        operating systems compatibility.
+      - The LDAP display name (ldapDisplayName) for this property is sAMAccountName.
+      - If ommitted the value is the same as C(name).
+      - Note that all computer SAMAccountNames needs to end with a $.
```
Comment (acozine) [inline lib/ansible/modules/windows/win_domain_computer.py]:
```suggestion
      - Note that all computer SAMAccountNames need to end with a $.
```

File: lib/ansible/modules/windows/win_domain_user.py [hunk 4]
```diff
@@ -21,59 +21,64 @@
...
         I(groups) and removed from any other groups.
+    type: str
     choices: [ add, remove, replace ]
     default: replace
   password:
     description:
-      - Optionally set the user's password to this (plain text) value. In order
-        to enable an account - I(enabled) - a password must already be
+      - Optionally set the user's password to this (plain text) value.
+      - In order to enable an account - I(enabled) - a password must already be
```
Comment (acozine) [inline lib/ansible/modules/windows/win_domain_user.py]:
```suggestion
      - To enable an account - I(enabled) - a password must already be
```

File: lib/ansible/modules/windows/win_environment.py [hunk 5]
```diff
@@ -21,25 +21,29 @@
...
   level:
     description:
     - The level at which to set the environment variable.
     - Use C(machine) to set for all users.
     - Use C(user) to set for the current user that ansible is connected as.
     - Use C(process) to set for the current process.  Probably not that useful.
-    choices: [ machine, user, process ]
+    type: str
     required: yes
+    choices: [ machine, user, process ]
```
Comment (acozine) [inline lib/ansible/modules/windows/win_environment.py]:
```suggestion
    choices: [ machine, process, user ]
```

File: lib/ansible/modules/windows/win_psexec.py [hunk 6]
```diff
@@ -78,22 +81,16 @@
...
     description:
     - Used to run the command at a different priority.
-    choices:
-    - background
-    - low
-    - belownormal
-    - abovenormal
-    - high
-    - realtime
+    choices: [ background, low, belownormal, abovenormal, high, realtime ]
```
Comment (acozine) [inline lib/ansible/modules/windows/win_psexec.py]:
```suggestion
    choices: [ abovenormal, background, belownormal, high, low, realtime ]
```

File: lib/ansible/modules/windows/win_uri.py [hunk 7]
```diff
@@ -78,7 +85,7 @@
       "application/json", then the JSON is additionally loaded into a key
       called C(json) in the dictionary results.
     type: bool
-    default: 'no'
+    default: n'
```
Comment (acozine) [inline lib/ansible/modules/windows/win_uri.py]:
```suggestion
    default: no
```

File: lib/ansible/modules/windows/win_wait_for.py [hunk 8]
```diff
@@ -95,40 +98,40 @@
...
     path: C:\temp\log.txt
 
-- name: wait until process complete is in the file before continuing
+- name: Wait until process complete is in the file before continuing
   win_wait_for:
     path: C:\temp\log.txt
     search_regex: process complete
 
-- name: wait until file if removed
+- name: Wait until file if removed
```
Comment (acozine) [inline lib/ansible/modules/windows/win_wait_for.py]:
```suggestion
- name: Wait until file is removed
```

**Comment by dagwieers** [comment]
@acozine Thanks for reviewing, all changes have been applied !

**Comment by dagwieers** [comment]
bot_skip

**Pull request merged**